nexus_constructor/data_model.py

```diff
@@ -50,6 +51,7 @@ class CylindricalGeometry(Geometry):
     The cylinder is assumed to have the center of its base located at the origin of the local coordinate system, and is
     described by the direction of its axis, its height, and radius.
     """
+    units = attr.ib(default="m", type=str)
```

Comment:
I've noticed one of the tests fails if the default unit is changed to anything other than metres. I'm not sure if this a problem or not.